[android][editor] 'No suitable category' info

map-per opened the pull request:
closes #6733

Added an explanation, that one can not find a category for everything in the list. The text guides users towards editing with other editors or using OSM Notes. (As soon as adding OSM notes with OM (#266) is possible we can link to that feature from here)


The intention behind the text is to prevent users from adding similar categories instead of the right ones. E.g. `drinking_water` instead of a `wastewater_plant` or `attraction` for everything (see #6733).
Moreover this avoids confusion, why e.g. address or building is not in the list.

The text is similar to the one suggested by @pastk in https://github.com/organicmaps/organicmaps/issues/6733#issuecomment-1833303387

|  |  |
|--------|--------|
| ![Screenshot_2024-03-30-19-04-08-596_app organicmaps debug](https://github.com/organicmaps/organicmaps/assets/79519062/2189be52-136a-484d-ac86-dc840c057aa0) | ![Screenshot_2024-03-30-19-05-19-342_app organicmaps debug](https://github.com/organicmaps/organicmaps/assets/79519062/49007a6e-364d-4284-b7bf-c3454cd221fd) | 

![Screenshot_2024-03-30-19-04-16-307_app organicmaps debug](https://github.com/organicmaps/organicmaps/assets/79519062/a013daa0-ccbb-4d23-9502-55997c79d323)

#### Details
- The information text is always shown below the list, so users see it after parsing the list
- FeatureCategoryFragment needed to be changed to keep the current behaviour of jumping to the top of the list when the search text is changed
- The textblock has a maxWidth to stay compact on big screens or in landscape mode

RedAuburn commented:
nice! it might be good if there was a button to add a note from that screen too :)

pastk commented:
Isn't the "category" word confusing in this context?
Maybe better to use e.g. "feature type"? (e.g. iD uses it)

biodranik commented:
The "category" wording comes from the search tab. Using a different word requires replacing it everywhere consistently.

Most users do not know what "lines, areas, tags, features, OSM" mean, and won't understand the text. Only those who are already familiar with OSM will read it properly.

map-per commented:
>Isn't the "category" word confusing in this context?
>Maybe better to use e.g. "feature type"? (e.g. iD uses it)

I think "category" is better here for users who don't know the more OSM specific / more technical term "feature type". 

>Most users do not know what "lines, areas, tags, features, OSM" mean, and won't understand the text. Only those who are already familiar with OSM will read it properly.

That's right. I updated the text to:

"Organic Maps does not support all categories. E.g. cities, railway stations, roads, lakes, or buildings can not be added. Please add such features directly via osm.org (use the Edit button in the top bar) or use a specialized OSM editing app."

map-per commented:
I updated the text and added translations

jimcarst reviewed (approved):
ok for nl

biodranik commented:
Should we recommend other OSM editors here and provide links to them?

muralito commented:
> Should we recommend other OSM editors here and provide links to them?

What about linking OSM Wiki https://wiki.openstreetmap.org/wiki/Editors ?

pastk commented:
> The "category" wording comes from the search tab. Using a different word requires replacing it everywhere consistently.

Search categories are different things. E.g. "where to eat" searches across multiple feature types.
I think this is the only place the "category" word is used in the editor (so easy to change):
![image](https://github.com/organicmaps/organicmaps/assets/18434508/319e5e04-e05f-45b8-8853-43c6c60f214a)


> I think "category" is better here for users who don't know the more OSM specific / more technical term "feature type".

IMHO "feature type" is more obvious / understandable.
E.g.
> "Organic Maps does not support all categories.

Categories of what? Its confusing without further refinement.

pastk commented on data/strings/strings.txt:
```diff
@@ -16382,6 +16382,143 @@
+
+  [editor_category_unsuitable_text1]
+    tags = android
+    en = Organic Maps does not support all categories. E.g. cities, railway stations, roads, lakes, or buildings can not be added. Please add such features directly via osm.org (use the Edit button in the top bar) or use a specialized OSM editing app.
```
Maybe first sentence like
`Organic Maps allows to add simple point features only, that means no towns, roads, lakes, building outlines, etc.`
?

> Please add such features directly via osm.org (use the Edit button in the top bar) or use 

Here I read it like there is an Edit button in OM's top bar that'll get me to the right place in osm.org... Maybe drop the `(use the Edit button in the top bar)` completely? or rephrase

biodranik replied:
"objects" then is clearer for newbies than a highly specific term "features".

pastk replied:
"objects", "map objects", "object types", etc. should be good indeed

NeatNit replied:
I second the suggestion for the first sentence, clarifying that only points are supported. Here's my slight edit to the suggestion:

```suggestion
    en = Organic Maps can only add point features, not lines or shapes. This means you cannot add roads, buildings, cities, railway stations, or lakes, among other things. Please add such features directly via osm.org (use the Edit button in the top bar of the website) or use a specialized OSM editing app.
```

I've also made roads and buildings first in the list of examples (they get the point across better than cities IMO) and clarified the top bar as noted.

Zverik commented:
First, I like the idea very much. The panel stands out, and serves a good introduction to where the data goes.

What'd I improve is shifting the focus from the website (we're on a phone remember) to apps. It should mention 2-3 most prominent mapping alternatives on the platform, different for iOS and Android: StreetComplete, Vespucci, Go Map, Every Door.

But that might make the panel too big, so we might consider making a dedicated page, and here just put a link. E.g. "To add or edit features of all types consider installing a dedicated app...".

Also, you can add notes with Organic Maps, so it should be a button, not a link.

vgrigoriu commented on android/app/src/main/res/values-ro/strings.xml:
```diff
@@ -488,6 +488,9 @@
 	<string name="editor_detailed_description">Modificările aduse hărții, sugerate de tine, vor fi trimise comunității OpenStreetMap. Descrie orice detalii suplimentare care nu pot fi modificate în Organic Maps.</string>
 	<string name="editor_more_about_osm">Mai multe despre OpenStreetMap</string>
 	<string name="editor_operator">Proprietar</string>
+	<string name="editor_category_unsuitable_title">Nu găsiți o categorie potrivită?</string>
+	<string name="editor_category_unsuitable_text1">Organic Maps nu acceptă toate categoriile. De exemplu, nu pot fi adăugate orașe, gări, drumuri, lacuri sau clădiri. Vă rugăm să adăugați astfel de caracteristici direct prin osm.org (utilizați butonul Edit din bara de sus) sau să folosiți o aplicație specializată de editare OSM.</string>
+	<string name="editor_category_unsuitable_text2">Alternativ, puteți utiliza și OSM Notes (osm.org/wiki/Notes) pentru rapoarte către comunitatea OSM.</string>
```
ro translation looks good.

pastk commented:
Re "categories" vs "feature types" vs "object types".


Using "objects" or "features" sounds good to me.

However when people will eventually proceed to osm.org (and we want them to do it) for wiki, beginner guide, editor, support chats, etc. they will have to re-learn to "map features" again.

So I'd rather start with the "osm standard term" "features" right away, IMHO its very understandable given the context and eventually there will be less confusion created by different terms usage.

My suggestion:

> Can't find a suitable feature type?
> 
> Organic Maps allows to add simple point features only, that means no towns, roads, lakes, building outlines, etc. Please add such features directly to OpenStreetMap.org (there is Edit button on the web site) or use a specialized OSM editing app like Every Door, StreetComplete or Vespucci.

(as OM doesn't support leaving standalone Notes yet then its better to drop this part IMO)

The "Category" caption (`[editor_edit_place_category_title]`) on the editing screen should be replaced with "Map Feature Type" or just "Feature Type".

And its better to replace the "Search" text inside the search field with a hint like `e.g. pharmacy or bus stop`.

biodranik commented:
If a user doesn't uninstall OM and visit OSM, then he/she will be able to learn anything.

Which of the definitions below fits the context?
<img width="710" alt="выява" src="https://github.com/organicmaps/organicmaps/assets/170263/385cc794-0c41-4391-851b-3e7bc6ba638f">

pastk commented:
If there is a concern that a "feature" is ambiguous / not specific enough then just use a "map feature"?
My understanding is that its a pretty standard term in mapping.

biodranik commented:
A detailed page on OM website may be better than everything else, because:
- OSM "Beginners guide" doesn't immediately show apps to install (and it may be too complex for grandparents willing to add some object on a map)
- The concept of "OSM is not OM" can be properly introduced
- We can use images
- We can list top editors and mention their difference
- We can mention ID too with a comment "use it on the desktop".

d4f5409d commented on android/app/src/main/res/values-hu/strings.xml:
```diff
@@ -490,6 +490,9 @@
 	<string name="editor_detailed_description">Javasolt változtatásait elküldjük az OpenStreetMap közösségnek. Írja le a további információkat, amelyek nem szerkeszthetők a Organic Maps-ben.</string>
 	<string name="editor_more_about_osm">További részletek az OpenStreetMap-ról</string>
 	<string name="editor_operator">Üzemeltető</string>
+	<string name="editor_category_unsuitable_title">Nem talál megfelelő kategóriát?</string>
```
```suggestion
	<string name="editor_category_unsuitable_title">Nem találja a megfelelő kategóriát?</string>
```

d4f5409d commented on android/app/src/main/res/values-hu/strings.xml:
```diff
@@ -490,6 +490,9 @@
 	<string name="editor_more_about_osm">További részletek az OpenStreetMap-ról</string>
 	<string name="editor_operator">Üzemeltető</string>
+	<string name="editor_category_unsuitable_title">Nem talál megfelelő kategóriát?</string>
+	<string name="editor_category_unsuitable_text1">Az Organic Maps nem támogat minden kategóriát. Pl. városok, vasútállomások, utak, tavak vagy épületek nem adhatók hozzá. Kérjük, adjon hozzá ilyen funkciókat közvetlenül az osm.org oldalon keresztül (használja a szerkesztés gombot a felső sávban), vagy használjon egy speciális OSM-szerkesztő alkalmazást.</string>
```
```suggestion
	<string name="editor_category_unsuitable_text1">Az Organic Maps sajnos nem támogat minden kategóriát. Pl. városok, vasútállomások, utak, tavak vagy épületek nem adhatók hozzá a térképhez. Kérjük, az ilyen típusú térképelemeket közvetlenül az osm.org oldalon keresztül adja hozzá (használja a szerkesztés gombot a felső sávban), vagy használjon egy speciális OSM-szerkesztő alkalmazást.</string>
```

d4f5409d reviewed (changes requested):
🇭🇺❌

map-per commented:
> A detailed page on OM website may be better than everything else, because:

Yes, that's the best. Should we add that page to the current website or wait until the new website is ready?

In terms of "categorie" vs "feature type" vs "object type" I don't have a strong opinion. But as you can't agree what is best I will keep "categories" for now, as it is currently used in OM.

#### Text:

Can't find a suitable category?

Organic Maps allows to add simple point features only, that means no towns, roads, lakes, building outlines, etc. Please add such features directly to OpenStreetMap.org. You can find a detailed step by step guide at organicmaps.app/osm.

Alternatively, you can also use OSM Notes (osm.org/wiki/Notes) for reports to the OSM Community.

d4f5409d commented:
If you ask me, adding infos to the website alone wouldn't help, if there wasn't referring link to it in the app, because newbies would be left with no strings attached how to get further into contributing to the map. So if we pace on putting this to the website, we should link that page in the app as well, in the style of the original idea (described to learn more at the dedicated Organic Maps page).

NeatNit commented on data/strings/strings.txt:
```diff
@@ -16382,6 +16382,143 @@
+    fa = نقشه های ارگانیک از همه دسته ها پشتیبانی نمی کند. به عنوان مثال. شهرها، ایستگاه های راه آهن، جاده ها، دریاچه ها یا ساختمان ها را نمی توان اضافه کرد. لطفاً چنین ویژگی هایی را مستقیماً از طریق osm.org اضافه کنید (از دکمه ویرایش در نوار بالا استفاده کنید) یا از یک برنامه ویرایش تخصصی OSM استفاده کنید.
+    fi = Organic Maps ei tue kaikkia luokkia. Esimerkiksi kaupunkeja, rautatieasemia, teitä, järviä tai rakennuksia ei voi lisätä. Lisää tällaiset ominaisuudet suoraan osm.org-sivuston kautta (käytä yläpalkin Muokkaa-painiketta) tai käytä erikoistunutta OSM-editointisovellusta.
+    fr = Organic Maps ne prend pas en charge toutes les catégories. Par exemple, les villes, les gares, les routes, les lacs ou les bâtiments ne peuvent pas être ajoutés. Ajoute ces éléments directement via osm.org (utilise le bouton Modifier dans la barre supérieure) ou utilise une application d'édition OSM spécialisée.
+    he = מפות אורגניות לא תומכות בכל הקטגוריות. לְמָשָׁל. לא ניתן להוסיף ערים, תחנות רכבת, כבישים, אגמים או מבנים. אנא הוסף תכונות כאלה ישירות דרך osm.org (השתמש בכפתור העריכה בסרגל העליון) או השתמש באפליקציית עריכה מיוחדת של OSM.
```
```suggestion
    he = האפליקציה Organic Maps יכולה להוסיף רק נקודות, לא קווים או צורות. למשל, לא ניתן להוסיף כבישים, מבנים, ערים, תחנות רכבת, או אגמים. ניתן להוסיף תכונות כאלה ישירות דרך osm.org (השתמשו בכפתור העריכה בסרגל העליון באתר) או דרך אפליקציית עריכה ייעודית של OSM.
```

I based this on the suggested rephrasing I posted above (which was based on pastk's suggestion) not the current commit. Here's a back-translation to English (via google translate):

> The Organic Maps app can only add points, not lines or shapes. For example, you cannot add roads, buildings, cities, train stations, or lakes. Such features can be added directly through osm.org (use the edit button in the top bar of the site) or through a dedicated OSM editor application.

NeatNit commented on data/strings/strings.txt:
```diff
@@ -16382,6 +16382,143 @@
+    fa = همچنین میu200cتوانید از یادداشتu200cهای OSM (osm.org/wiki/Notes) برای گزارشu200cهایی به انجمن OSM استفاده کنید.
+    fi = Vaihtoehtoisesti voit myös käyttää OSM Notesia (osm.org/wiki/Notes) OSM-yhteisölle tehtäviin raportteihin.
+    fr = Tu peux aussi utiliser OSM Notes (osm.org/wiki/Notes) pour les rapports destinés à la communauté OSM.
+    he = לחלופין, אתה יכול גם להשתמש ב-OSM Notes (osm.org/wiki/Notes) לדיווחים לקהילת OSM.
```
Edit for `he`:
```suggestion
    he = לחלופין, ניתן להוסיף הערות למפה (osm.org/wiki/Notes) כדי לדווח לקהילת OSM.
```

Q: Does Organic Maps allow users to add notes to the map? If not I don't see the value of this string, or at least it needs to be clarified that OM doesn't support this either.

pastk commented:
> 
> In terms of "categorie" vs "feature type" vs "object type" I don't have a strong opinion. But as you can't agree what is best I will keep "categories" for now, as it is currently used in OM.

Actually the original text from this PR used "features" also as in `Please add such features directly via ...` :) Its just comes naturally, isn't it?

> #### Text:
> 
> Can't find a suitable category?
> 
> Organic Maps allows to add simple point features only, that means no towns, roads, lakes, building outlines, etc. Please add such features directly to OpenStreetMap.org. You can find a detailed step by step guide at organicmaps.app/osm.
> 
> Alternatively, you can also use OSM Notes (osm.org/wiki/Notes) for reports to the OSM Community.

Here its a mix of categories and features again. Let's pick just one then!

matheusgomesms commented on data/strings/strings.txt:
```diff
@@ -16382,6 +16382,143 @@
+    nb = Finner du ikke en passende kategori?
+    nl = Kun je geen geschikte categorie vinden?
+    pl = Nie możesz znaleźć odpowiedniej kategorii?
+    pt = Não consegues encontrar uma categoria adequada?
+    pt-BR = Você não consegue encontrar uma categoria adequada?
```
```suggestion
    pt = Não encontra uma categoria adequada?
    pt-BR = Você não encontra uma categoria adequada?
```

matheusgomesms commented on data/strings/strings.txt:
```diff
@@ -16382,6 +16382,143 @@
+    nb = Organic Maps støtter ikke alle kategorier. Det er f.eks. ikke mulig å legge til byer, jernbanestasjoner, veier, innsjøer eller bygninger. Legg til slike funksjoner direkte via osm.org (bruk Rediger-knappen i topplinjen) eller bruk en spesialisert OSM-redigeringsapp.
+    nl = Organic Maps ondersteunt niet alle categorieën. Je kunt bijvoorbeeld geen steden, stations, wegen, meren of gebouwen toevoegen. Voeg zulke functies rechtstreeks toe via osm.org (gebruik de knop Bewerken in de bovenste balk) of gebruik een gespecialiseerde OSM-bewerkingsapp.
+    pl = Organic Maps nie obsługuje wszystkich kategorii. Na przykład nie można dodawać miast, stacji kolejowych, dróg, jezior lub budynków. Dodaj takie elementy bezpośrednio przez osm.org (użyj przycisku Edytuj na górnym pasku) lub skorzystaj ze specjalistycznej aplikacji do edycji OSM.
+    pt = Os Mapas Orgânicos não suportam todas as categorias. Por exemplo, não podes adicionar cidades, estações de comboio, estradas, lagos ou edifícios. Por favor, adiciona essas características diretamente através de osm.org (usa o botão Editar na barra superior) ou usa uma aplicação de edição OSM especializada.
+    pt-BR = O Organic Maps não é compatível com todas as categorias. Por exemplo, não é possível adicionar cidades, estações de trem, estradas, lagos ou edifícios. Adicione esses recursos diretamente em osm.org (use o botão Editar na barra superior) ou use um aplicativo de edição OSM especializado.
```
```suggestion
    pt = O Organic Maps não suporta todas as categorias. Por exemplo, não é possível adicionar cidades, estações de comboio, estradas, lagos ou edifícios. Adicione esses elementos diretamente em osm.org (use o botão Editar na barra superior) ou use uma aplicação especializada em edição do OSM.
    pt-BR = O Organic Maps não suporta todas as categorias. Por exemplo, não é possível adicionar cidades, estações de trem, estradas, lagos ou edifícios. Adicione esses elementos diretamente em osm.org (use o botão Editar na barra superior) ou use um aplicativo especializado em edição do OSM.
```

matheusgomesms reviewed (changes requested):
PT and PT-BR review.

map-per commented:
Ok, so now the text is:

Can't find a suitable feature type?

Organic Maps allows to add simple point features only, that means no towns, roads, lakes, building outlines, etc. Please add such features directly to OpenStreetMap.org. You can find a detailed step by step guide at organicmaps.app/faq/editing/advanced-map-editing.

Alternatively, you can also use OSM Notes for reports to the OSM Community. Visit osm.org/wiki/Notes for further details.

biodranik commented:
Thanks! What about a more simplified version?

---
Can't find a suitable object type?

Towns, roads, lakes, buildings, and other objects can be added at [OpenStreetMap.org](https://www.openstreetmap.org) or in other map editors. Check [our guide](https://organicmaps.app/faq/editing/advanced-map-editing) or leave a note for the OSM community volunteers by long-tapping the place and selecting "Edit Place".

---

The link should lead to a newbie-friendly FAQ page, which, in turn, should lead to a more advanced web page.

map-per commented:
First of all: Thanks for providing a full and written out alternative suggestion!

But with that text it's not really clear that you need osm.org only for some things, but can edit most features within OM.
And telling people who want to report that something is missing to "long-tapping the place" is rather confusing.

I think its also better to visually separate osm.org editing and note, so we can later add an "Add a note" button below the second section.

Let's stick with the text and not overcomplicate things.

pastk commented:
> or leave a note for the OSM community volunteers by long-tapping the place and selecting "Edit Place".

but it doesn't always work like that (e.g. can't do it on an "unknown place")..
I think first an independent osm note adding functionality should be improved and then we'll be able to add a direct button/link.

> Ok, so now the text is:
> 

Sounds good!
For me either version of the wording and terms used (category/feature/object - up to you @map-per ) is fine and is a good improvement already.

Just please make sure its used consistently so `[editor_edit_place_category_title]` in strings.txt might need to be changed.

map-per commented:
I removed the paragraph about OSM Notes, shortened the URL text, and updated the text.

<img src="https://github.com/organicmaps/organicmaps/assets/79519062/6c025919-2c1c-4923-b282-c14c1362ecdd" width="400">

jimcarst commented:
nl is ok

matheusgomesms commented on data/strings/strings.txt:
```diff
@@ -16401,6 +16401,96 @@
+    nb = Organic Maps tillater kun å legge til enkle punktkategorier, det vil si ingen byer, veier, innsjøer, bygningsomriss osv. Vennligst legg til slike kategorier direkte til <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Sjekk vår <a href="https://organicmaps.app/faq/editing/advanced-map-editing">guide</a> for detaljerte trinnvise instruksjoner.
+    nl = Met Organic Maps kun je alleen eenvoudige puntcategorieën toevoegen, dus geen steden, wegen, meren, contouren van gebouwen, enzovoort. Voeg zulke categorieën direct toe aan <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Bekijk onze <a href="https://organicmaps.app/faq/editing/advanced-map-editing">gids</a> voor gedetailleerde stap voor stap instructies.
+    pl = Organic Maps pozwala na dodawanie tylko prostych kategorii punktów, co oznacza brak miast, dróg, jezior, obrysów budynków itp. Dodaj takie kategorie bezpośrednio do <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Sprawdź nasz <a href="https://organicmaps.app/faq/editing/advanced-map-editing">przewodnik</a>, aby uzyskać szczegółowe instrukcje krok po kroku.
+    pt = Organic Maps permite-te adicionar apenas categorias de pontos simples, ou seja, nada de cidades, estradas, lagos, contornos de edifícios, etc. Por favor, adiciona essas categorias diretamente ao <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Consulta o nosso <a href="https://organicmaps.app/faq/editing/advanced-map-editing">guia</a> para instruções detalhadas passo a pass ...
+    pt-BR = O Organic Maps permite que você adicione apenas categorias de pontos simples, o que significa que não há cidades, estradas, lagos, contornos de edifícios, etc. Adicione essas categorias diretamente ao <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Consulte nosso <a href="https://organicmaps.app/faq/editing/advanced-map-editing">guia</a> para obter instruções detalhadas ...
```
```suggestion
    pt = O Organic Maps permite-lhe adicionar apenas categorias de pontos simples, ou seja, nada de cidades, estradas, lagos, contornos de edifícios etc. Por favor, adicione essas categorias diretamente ao <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Consulte o nosso <a href="https://organicmaps.app/faq/editing/advanced-map-editing">guia</a> para instruções detalhadas passo a passo.
    pt-BR = O Organic Maps permite que você adicione apenas categorias de pontos simples, o que significa que não há cidades, estradas, lagos, contornos de edifícios etc. Por favor, adicione essas categorias diretamente no <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Consulte nosso <a href="https://organicmaps.app/faq/editing/advanced-map-editing">guia</a> para obter instruções detalhadas passo a passo.
```

Jean-BaptisteC reviewed (approved):
✅ Pixel 6 - Android 14

map-per commented:
I updated the translations, so this is ready for merge now

NeatNit commented on data/strings/strings.txt:
```diff
@@ -16446,6 +16446,96 @@
+    fa = Organic Maps اجازه میu200cدهد فقط دستهu200cهای نقطهu200cای ساده را اضافه کند، یعنی هیچ شهر، جاده، دریاچه، طرح کلی ساختمان و غیره وجود ندارد. لطفاً این دستهu200cها را مستقیماً به <a href="https://www.openstreetmap.org">OpenStreetMap.org</a> اضافه کنید. برای دستورالعملu200cهای گام به گام، <a href="https://organicmaps.app/faq/editing/advanced-map-editing">راهنمای</a> ما را بررسی کنید.
+    fi = Organic Maps mahdollistaa vain yksinkertaisten pisteluokkien lisäämisen, eli ei kaupunkeja, teitä, järviä, rakennusten ääriviivoja jne. Lisää tällaiset luokat suoraan <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Tarkista <a href="https://organicmaps.app/faq/editing/advanced-map-editing">oppaastamme</a> yksityiskohtaiset ohjeet vaihe vaiheelta.
+    fr = Organic Maps ne permet d'ajouter que des catégories de points simples, c'est-à-dire pas de villes, de routes, de lacs, de contours de bâtiments, etc. Merci d'ajouter ces catégories directement sur <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Consulte notre <a href="https://organicmaps.app/faq/editing/advanced-map-editing">guide</a> pour obtenir des instructions détaillé ...
+    he = מפות אורגניות מאפשרות להוסיף קטגוריות נקודות פשוטות בלבד, כלומר ללא עיירות, כבישים, אגמים, קווי מתאר וכו'. אנא הוסף קטגוריות כאלה ישירות ל<a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. עיין ב<a href="https://organicmaps.app/faq/editing/advanced-map-editing">מדריך</a> שלנו לקבלת הוראות צעד אחר צעד מפורטות.
```
```suggestion
    he = באמצעות Organic Maps ניתן להוסיף קטגוריות נקודה פשוטות בלבד, כלומר לא ניתן להוסיף עיירות, כבישים, אגמים, מבנים וכו'. אנא הוסיפו קטגוריות כאלה ישירות ל-<a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. עיינו <a href="https://organicmaps.app/faq/editing/advanced-map-editing">במדריך</a> שלנו לקבלת הוראות מפורטות.
```

NeatNit reviewed (changes requested):
No, the translations are bad because it's a whole paragraph (with embedded links, no less) and needs human review. @organicmaps/translations

muralito commented on android/app/src/main/res/values-es/strings.xml:
```diff
@@ -504,6 +504,8 @@
 	<string name="editor_more_about_osm">Más acerca de OpenStreetMap</string>
 	<string name="editor_operator">Operador</string>
+	<string name="editor_category_unsuitable_title">¿No halla una categoría adecuada?</string>
+	<string name="editor_category_unsuitable_text">Organic Maps sólo permite añadir categorías de puntos simples, es decir, no ciudades, carreteras, lagos, contornos de edificios, etc. Por favor, añade dichas categorías directamente a <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Consulta nuestra <a href="https://organicmaps.app/faq/editing/advanced-map-editing">guía</a> para obtene ...
```
Prefiero:

<string name="editor_category_unsuitable_text">Organic Maps permite añadir únicamente categorías de puntos sencillos, es decir, no ciudades, carreteras, lagos, contornos de edificios, etc. Por favor, añade dichas categorías directamente a <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Consulta nuestra <a href="https://organicmaps.app/faq/editing/advanced-map-editing">guía</a> para obtener instrucciones detalladas paso a paso.</string>

gpesquero replied:
@muralito: Your suggestion looks good, but we have to use the *formal* language version: 

Organic Maps permite añadir únicamente categorías de puntos sencillos, es decir, no ciudades, carreteras, lagos, contornos de edificios, etc. Por favor, añad**a** dichas categorías directamente a [OpenStreetMap.org](https://www.openstreetmap.org). Consult**e** nuestra [guía](https://organicmaps.app/faq/editing/advanced-map-editing) para obtener instrucciones detalladas paso a paso.

pastk commented:
@biodranik is it good enough to merge?

biodranik commented on data/strings/strings.txt:
```diff
@@ -16446,6 +16446,96 @@
+
+  [editor_category_unsuitable_text]
+    tags = android
+    en = Organic Maps allows to add simple point categories only, that means no towns, roads, lakes, building outlines, etc. Please add such categories directly to <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Check our <a href="https://organicmaps.app/faq/editing/advanced-map-editing">guide</a> for detailed step by step instructions.
```
There is already "Categories?" mentioned in the title, no need to mention it here. We can actually include all 3 forms: categories + object types + features, bingo!
Also a few more improvements:


`en = Organic Maps allows adding only simple object types, which means no towns, roads, lakes, building outlines, etc. Please add or edit more complex features at <a href="https://www.openstreetmap.org/">OpenStreetMap.org</a>. Check our <a href="https://organicmaps.app/faq/editing/advanced-map-editing/">guide</a> for detailed step-by-step instructions and other recommended map editing apps.`

map-per replied:
> We can actually include all 3 forms: categories + object types + features, bingo!

Please not, that's just confusing

map-per replied:
For the future it would be good if you could at some point indicate that the English string is good and ready for translation reviews. It's not good that the translator now have to translate similar string over and over again.

biodranik replied:
Sorry, it is not enough time to manage that amount of PRs alone. En texts can be pre-approved before anything else next time, even in the chat.

pastk replied:
another option is make separate translation PRs after the main one is merged (incl. en strings) - it'll reduce amount of non-relevant notifications for translators too

the downside is that its extra work and surely it'll get forgotten sometimes

pastk replied:
(and with Weblate it'll be a non-issue)

map-per commented:
Updated he and es strings

biodranik commented on data/strings/strings.txt:
```diff
@@ -16446,6 +16446,96 @@
+    pt = O Organic Maps permite-lhe adicionar apenas categorias de pontos simples, ou seja, nada de cidades, estradas, lagos, contornos de edifícios etc. Por favor, adicione essas categorias diretamente ao <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Consulte o nosso <a href="https://organicmaps.app/faq/editing/advanced-map-editing">guia</a> para instruções detalhadas passo a pa ...
+    pt-BR = O Organic Maps permite que você adicione apenas categorias de pontos simples, o que significa que não há cidades, estradas, lagos, contornos de edifícios etc. Por favor, adicione essas categorias diretamente no <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Consulte nosso <a href="https://organicmaps.app/faq/editing/advanced-map-editing">guia</a> para obter instruções  ...
+    ro = Organic Maps permite adăugarea doar a unor categorii de puncte simple, ceea ce înseamnă că nu există orașe, drumuri, lacuri, contururi de clădiri, etc. Vă rugăm să adăugați astfel de categorii direct la <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Consultați <a href="https://organicmaps.app/faq/editing/advanced-map-editing">ghidul nostru</a> pentru instrucțiuni detaliat ...
+    ru = Organic Maps позволяет добавлять только простые категории точек, то есть никаких городов, дорог, озер, контуров зданий и так далее. Пожалуйста, добавляй такие категории непосредственно на <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Ознакомься с нашим <a href="https://organicmaps.app/faq/editing/advanced-map-editing">руководством</a> для получения подробных пошаговых ин ...
```
Organic Maps позволяет добавлять на карту только простые типы объектов, то есть никаких городов, дорог, озер, контуров зданий. Пожалуйста, добавляйте такие категории на сайте <a href="https://www.openstreetmap.org/">OpenStreetMap.org</a>. Так же рекомендуем ознакомиться с нашими <a href="https://organicmaps.app/faq/editing/advanced-map-editing/">подробными пошаговыми инструкциями и другими приложениями для редактирования карты</a>.

biodranik commented on data/strings/strings.txt:
```diff
@@ -16446,6 +16446,96 @@
+    af = Organiese kaarte laat toe om slegs eenvoudige puntkategorieë by te voeg, dit beteken geen dorpe, paaie, mere, gebouomlynings, ens. Voeg asseblief sulke kategorieë direk by <a href="https://www.openstreetmap.org">OpenStreetMap.org< /a>. Gaan ons <a href="https://organicmaps.app/faq/editing/advanced-map-editing">gids</a> na vir gedetailleerde stap-vir-stap-instruksies.
+    ar = تسمح الخرائط العضوية بإضافة فئات نقاط بسيطة فقط، وهذا يعني عدم وجود بلدات وطرق وبحيرات ومخططات للمباني وما إلى ذلك. يرجى إضافة هذه الفئات مباشرة إلى <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. راجع موقعنا <a href="https://organicmaps.app/faq/editing/advanced-map-editing">دليل</a> للحصول على تعليمات مفصلة خطوة بخطوة.
+    az = Üzvi Xəritələr yalnız sadə nöqtə kateqoriyaları əlavə etməyə imkan verir, bu o deməkdir ki, şəhərlər, yollar, göllər, bina konturları və s. yoxdur. Lütfən, belə kateqoriyaları birbaşa <a href="https://www.openstreetmap.org">OpenStreetMap.org</a> saytına əlavə edin. Ətraflı addım-addım təlimatlar üçün <a href="https://organicmaps.app/faq/editing/advanced-map-editing">bələdçimizi</a> yoxla ...
+    be = Organic Maps дазваляе дадаваць толькі простыя катэгорыі кропак, што азначае адсутнасць гарадоў, дарог, азёр, абрысаў будынкаў і г.д. Калі ласка, дадавайце такія катэгорыі непасрэдна на <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Праверце наш <a href="https://organicmaps.app/faq/editing/advanced-map-editing">кіраўніцтва</a> для атрымання падрабязных пакрокавых інструкцы ...
```
```suggestion
    be = Organic Maps дазваляе дадаваць на мапу толькі простыя тыпы аб'ектаў, гэта значыць без гарадаў, дарог, азёр і будынкаў. Калі ласка, дадайце такія катэгорыі на сайце <a href="https://www.openstreetmap.org/">OpenStreetMap.org </a>. Мы таксама рэкамендуем азнаёміцца ​​з <a href="https://organicmaps.app/faq/editing/advanced-map-editing/">падрабязнымі інструкцыямі і іншымі прыкладаннямі для рэдагавання мапы</a>.
```

biodranik commented on data/strings/strings.txt:
```diff
@@ -16446,6 +16446,96 @@
+    sw = Ramani za Kikaboni huruhusu kuongeza kategoria rahisi pekee, hiyo inamaanisha hakuna miji, barabara, maziwa, muhtasari wa majengo, n.k. Tafadhali ongeza kategoria kama hizo moja kwa moja kwenye <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Angalia <a href="https://organicmaps.app/faq/editing/advanced-map-editing">mwongozo wetu</a> kwa maagizo ya kina ya hatua kwa hatua.
+    th = แผนที่ทั่วไปอนุญาตให้เพิ่มหมวดหมู่จุดธรรมดาเท่านั้น ซึ่งหมายความว่าไม่มีเมือง ถนน ทะเลสาบ โครงร่างอาคาร ฯลฯ โปรดเพิ่มหมวดหมู่ดังกล่าวลงใน <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. ดู<a href="https://organicmaps.app/faq/editing/advanced-map-editing">คำแนะนำ</a> ของเราเพื่อดูคำแนะนำโดยละเอียดทีละขั้นตอน
+    tr = Organik Haritalar yalnızca basit nokta kategorileri eklemeye izin verir, yani kasaba, yol, göl, bina ana hatları vb. yoktur. Lütfen bu tür kategorileri doğrudan <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>'a ekleyin. Adım adım ayrıntılı talimatlar için <a href="https://organicmaps.app/faq/editing/advanced-map-editing">rehberimize</a> göz atın.
+    uk = Organic Maps дозволяє додавати лише прості категорії точок, тобто без міст, доріг, озер, контурів будівель тощо. Будь ласка, додавайте такі категорії безпосередньо на <a href="https://www.openstreetmap.org">OpenStreetMap.org</a>. Перевірте наш <a href="https://organicmaps.app/faq/editing/advanced-map-editing">посібник</a> для отримання детальних покрокових інструкцій.
```
```suggestion
    uk = Organic Maps дозволяють додавати до мапи лише прості типи об'єктів, тобто без міст, доріг, озер та контурів будівель. Будь ласка, додайте такі категорії на сайті <a href="https://www.openstreetmap.org/">OpenStreetMap.org</a>. Ми також рекомендуємо ознайомитися із <a href="https://organicmaps.app/faq/editing/advanced-map-editing/">детальними інструкціями та іншими додатками для редагування карти</a>.
```

map-per commented on android/app/src/main/java/app/organicmaps/editor/FeatureCategoryFragment.java:
```diff
@@ -55,6 +62,10 @@ protected void onTextChanged(String query)
+    mScrollView = view.findViewById(R.id.nested_scroll_view);
+
+    categoryUnsuitableText = view.findViewById(R.id.editor_category_unsuitable_text);
+    categoryUnsuitableText.setMovementMethod(LinkMovementMethod.getInstance());
```
No, that's not necessary, I will remove it

pastk commented:
@map-per could you please include the last translation changes and we'll merge to take it into the release?

map-per commented:
:+1: Updated the translations and used a local variable for `categoryUnsuitableText`

biodranik reviewed (approved):
Thanks! @kirylkaveryn can you please add a similar hint with the same strings to iOS?

biodranik commented:
And many thanks to everyone who participated, community is the power! 🚀
